static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -215,39 +262,13 @@ const MutationListItem = styled('li')`
   &:only-of-type::after {
     height: 0;
   }
-`;
-
-const MutationContent = styled('div')`
-  overflow: hidden;
-  width: 100%;
-
-  display: flex;
-  flex-direction: column;
-  gap: ${space(1)};
-`;
-
-const MutationDetailsContainer = styled('div')`
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-start;
-  flex-grow: 1;
-`;
 
-/**
- * Taken `from events/interfaces/.../breadcrumbs/types`
- */
-const IconWrapper = styled('div')<Required<Pick<SVGIconProps, 'color'>>>`
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  width: 24px;
-  min-width: 24px;
-  height: 24px;
-  border-radius: 50%;
-  color: ${p => p.theme.white};
-  background: ${p => p.theme[p.color] ?? p.color};
-  box-shadow: ${p => p.theme.dropShadowLightest};
-  z-index: 2;
+  ${MutationDetailsContainer} {
+    opacity: ${p => (p.hasOccurred ? 1 : 0.5)};
```

Comment:
is opacity the only way to do this? how will we solve the problem when the icon colors get updated?

Reply:
the existing console implementation does not use opacity. they should be using the same techniques afaik.

Reply:
The lighter colored icon is itself opaque. So we will need a lighter icon color that doesn't have a lesser opacity